overlord/snapstate/snapstate.go

```diff
@@ -309,6 +362,10 @@ func Remove(s *state.State, name string) (*state.TaskSet, error) {
 		return nil, fmt.Errorf("snap %q is not removable", name)
 	}
 
+	if err := checkChangeConflict(s, name); err != nil {
```

Comment:
sorry, I wasn't clear I don't think it can be moved this much, Info() could give spurious errors if used while the snap is being touched I think